test/system/registered_applications_test.rb

```diff
@@ -0,0 +1,9 @@
+require "application_system_test_case"
```

Comment:
That was generated automatically. I'm not sure what is the distinction between the `test` directory and the `spec` directory.